src/sentry/mail/adapter.py

```diff
@@ -321,12 +323,17 @@ def notify(self, notification, target_type, target_identifier=None, **kwargs):
                         commit_data["subject"] = commit_data["message"].split("\n", 1)[0]
                         commits[commit["id"]] = commit_data
 
+        project_plugins = plugins.for_project(project, version=1)
+        organization_integrations = Integration.objects.filter(organizations=org).first()
+        has_integrations = True if project_plugins or organization_integrations else False
```

Comment:
I think this could be clearer as `!!(project_plugins or organization_integrations)` or
`bool(project_plugins or organization_integrations)`.